lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1196,9 +1195,9 @@ LogicalResult DotOperandEncodingAttr::verify(
 
   if (auto parentAttr = mlir::dyn_cast<AMDWmmaEncodingAttr>(parent)) {
     if (kWidth != 16 && parentAttr.getVersion() == 1 ||
-        kWidth != 8 && parentAttr.getVersion() == 2)
+        kWidth != 8 && kWidth != 16 && parentAttr.getVersion() == 2)
```

Comment:
Has ``kWidth == 16`` been already supported for WMMAv2?

Reply:
@giuseros mentioned that he needs k dimension size = 32, either for small data types, or for kpack=2. I am not sure what is the exact reason though.

Reply:
I dont mind